v1/pipeline/stages/cd/service-yaml.yaml

```diff
@@ -1,18 +1,16 @@
 title: ServiceYaml
 type: object
 required:
-- identifier
-- name
+  - ref
 properties:
-  description:
+  inputs:
+    oneOf:
+      - $ref: ../../common/json-node.yaml
```

Comment:
@shalini-agr are we taking like this jsonNode?

Reply:
we should use this instead
`inputs:
    type: object
    description: inputs to be used in the linked template
    additionalProperties:
      anyOf:
        - string
        - array
        - number
        - object
        - boolean
    propertyNames:
      pattern: "^[a-zA-Z_][0-9a-zA-Z_\\.$-]{0,127}$"`

Reply:
Thanks @shalini-agr. I think for every entity we are gonna same inputs schema. I have created this file to be used atleast in cd context. cd-inputs.yaml

Reply:
@tchaurasiya @shalini-agr can we create one common ref file for this inputs?

Reply:
discussed offline. moved to common package for every ones usages.